finality-provider/service/fp_manager.go

```diff
@@ -247,6 +246,11 @@ func (fpm *FinalityProviderManager) StartFinalityProvider(fpPk *bbntypes.BIP340P
 	}
 
 	if err := fpm.addFinalityProviderInstance(fpPk, passphrase); err != nil {
+		if errors.Is(err, ErrFinalityProviderJailed) {
+			fpm.logger.Error("failed to start finality provider", zap.Error(err))
+			// do not return error
```

Comment:
why not? :thinking:

Reply:
Same question here. This seems like an error to me

Reply:
The point of not returning error is that we still want to start the app even if we fail to start the fp instance due to fp jailed. But I agree that we should not handle the error here. I changed to handle the error in the application level in [0eb53b3](https://github.com/babylonlabs-io/finality-provider/pull/78/commits/0eb53b366b0743a9334004d88901751e8a6d215e)